src/modules/task-show.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable import/no-cycle */
+import { deleteTask, storageUpdate } from './localstorage.js';
+import { inputNewTasks, buttonsDeactive } from '../index.js';
+
+export const toDoDivTask = document.getElementById('sect__one__cont__items__item__two');
+const deletToDoTasks = document.querySelector('.item__three__btn__del');
+let clearToDoTasks;
+let modifyToDoTasks;
+let counter;
+let toDotasksupdate = '';
+
+/* || FUNCTION TO DISPLAY THE TASKS */
+export const showToDoTasks = () => {
+  if (Object.keys(localStorage).length > 0) {
+    toDoDivTask.style.display = 'inline-block';
+  } else {
+    toDoDivTask.style.display = 'none';
+  }
+  /* || Clear the tasks */
+  toDoDivTask.innerHTML = '';
+  /* || Fetch All The Keys in local storage */
+  let toDoTasks = Object.keys(localStorage);
+  toDoTasks = toDoTasks.sort();
+
+  toDoTasks.forEach((key) => {
+    /* || Get all values */
+    const toDoValue = localStorage.getItem(key);
+
+    const toDoInnerDiv = document.createElement('p');
+    toDoInnerDiv.classList.add('p__cont__task');
+    toDoInnerDiv.setAttribute('id', key);
+    toDoInnerDiv.innerHTML = `<p id="todo__name">${key.split('_')[1]}</p>`;
+    /* || localstorage would store boolean as string so we parse it to boolean back */
+    const buttonModify = document.createElement('button');
+    buttonModify.classList.add('edit__btn');
+    buttonModify.innerHTML = '<i class="fa-solid fa-pen-to-square"></i>';
+    if (!JSON.parse(toDoValue)) {
+      buttonModify.style.visibility = 'visible';
+    } else {
+      buttonModify.style.visibility = 'hidden';
+      toDoInnerDiv.classList.add('completedTask');
+    }
+    toDoInnerDiv.appendChild(buttonModify);
+    toDoInnerDiv.innerHTML += '<button class="del__btn"><i class="fa-solid fa-trash"></i></button>';
+    toDoDivTask.appendChild(toDoInnerDiv);
+  });
+  /* || tasks completed */
+  toDoTasks = document.querySelectorAll('.p__cont__task');
+  toDoTasks.forEach((element) => {
+    element.onclick = () => {
+      /*  || local storage update */
+      if (element.classList.contains('completedTask')) {
+        storageUpdate(element.id.split('_')[0], element.innerText, false);
+      } else {
+        storageUpdate(element.id.split('_')[0], element.innerText, true);
+      }
+    };
+  });
```

Comment:
- [ ] Good job on crafting the functionality to set an item to completed 👏 and kudos on using comments for describing the block of codes. However, from the project requirements, **The functionality for marking the tasks as completed should be placed in its own file and then later import where you want to call it** 
So please create a separate file and then place the following functionality into it. Later, you will import it just as you import other modules 😉